omnibus/config/chefdk_overrides.rb

```diff
@@ -0,0 +1,23 @@
+override :berkshelf, :version => "v4.3.1"
```

Comment:
It would be nice if we used something like `gem_override` here so it does not get confused with `override` in the project file.